sdk/tests/conformance/extensions/ext-frag-depth.html

```diff
@@ -106,27 +106,40 @@
 var gl = wtu.create3DContext(canvas);
 var ext = null;
 
-if (!gl) {
-    testFailed("WebGL context does not exist");
-} else {
-    testPassed("WebGL context exists");
+// Run all tests once.
+runAllTests();
 
-    runShaderTests(false);
+// Run all tests against with a new context to test for any cache issues.
+debug("");
+debug("Testing new context to catch cache errors");
+var canvas2 = document.getElementById("canvas2");
+gl = wtu.create3DContext(canvas2);
```

Comment:
We can eliminate both the "canvas" and "canvas2" elements and just call "wtu.create3DContext()" for both. Could you make that cleanup in these tests?